<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectFoodControlUI</class>
 <widget class="QWidget" name="SelectFoodControlUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SelectFoodControl</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblFoodListTitle">
     <property name="text">
      <string>Search Results:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lstResults">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Add Amount:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtAmount"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbUnit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>To Meal:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbMeal">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
